srunner/scenariomanager/scenarioatomics/atomic_criteria.py

```diff
@@ -1020,9 +1020,15 @@ def update(self):
         if location is None:
             return new_status
 
-        # Check if outside route lanes
-        self._is_outside_driving_lanes(location)
-        self._is_at_wrong_lane(location)
+        # Deactivate by other components
+        activate = py_trees.blackboard.Blackboard().get('AC_SwitchOutsideRouteLanesTest')
+        if activate is not None and activate is False:
```

Comment:
I'd recommend against this. Better to just do the check [here](https://github.com/carla-simulator/scenario_runner/blob/leaderboard-2.0/srunner/scenariomanager/scenarioatomics/atomic_criteria.py#L1051)

Reply:
Moved this part of code to https://github.com/carla-simulator/scenario_runner/blob/leaderboard-2.0/srunner/scenariomanager/scenarioatomics/atomic_criteria.py#L1051